src/libraries/System.Private.CoreLib/src/System/Reflection/IReflect.cs

```diff
@@ -49,11 +49,21 @@ public interface IReflect
         PropertyInfo[] GetProperties(BindingFlags bindingAttr);
 
         // Return an array of members which match the passed in name.
-        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields |
```

Comment:
Why don't these two use `Type.MemberAnnotations`?

Reply:
I think C# doesn't allow constants in interfaces because a constant is an implementation detail, so does not belong in a type that only defines behavior. It would have to be wrapped into an enum or struct. The IReflect file basically didn't have any knowledge of what the constant was, so I just use all the values.

Reply:
I don't understand the reasoning. You can definitely reference a constant in the `Type` class from this file.

Reply:
Sorry, apparently I just had to call System.Type.GetAllMembers and not just GetAllMembers. I thought it wasn't finding it because of interface language-specific behavior